stripe/util.py

```diff
@@ -1,10 +1,24 @@
+from __future__ import print_function
```

Comment:
The project already uses `2to3`, so technically you could remove this import and just use the `print` statement -- it will automatically be converted to the equivalent function in Python 3.

(I would personally prefer that we use `__future__` and/or `six` and not rely on automatic code conversion though.)